src/main/java/frc/robot/subsystems/amp/AmpIO.java

```diff
@@ -0,0 +1,64 @@
+package frc.robot.subsystems.amp;
+
+import org.littletonrobotics.junction.AutoLog;
+
+import frc.robot.subsystems.shooter.ShooterIO.ShooterIOInputs;
+
+public interface AmpIO {
+
+  @AutoLog
+  public static class AmpIOInputs {
+    public double ampBarCurrentPosition = 0.0;
```

Comment:
In my opinion you should specify the component of the subsystem you're talking about here and call them spinner and pivot because and bar spinners aren't really shooters.